### DISCUSSION TITLE
Visual C++ Redistributable for Visual Studio 2010

### DISCUSSION THREAD

**Discussion by lampra1** [SBFspot/SBFspot#613] [General]
Could you please provide a hint in the wiki that "Visual C++ Redistributable for Visual Studio 2010" is needed for windows?
This is needed for 64bit windows (at least 10 & 11) and for me it was difficult to spot.
Ideally, the download page for "Visual C++ Redistributable for Visual Studio 2010" could be included in the wiki.
https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x86.exe
This download page is available in:
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2010-vc-100-sp1-no-longer-supported

**Comment by sbf-** [comment]
SBFspot for windows is built with Visual Studio 2015, so Visual C++ Redistributable for Visual Studio 2015 (32-bit) should be installed.
